_layouts/quickstart.html

```diff
@@ -0,0 +1,30 @@
+---
+pagename: Quickstart
+latestversion: "1.14"
+---
+{% include header.html %}
+<div id="wrap">
+	<div class="container">
+	  <div class="row">
+			{% capture versionmaj %}{{ page.version | split: "." | first | times: 100 }}{% endcapture %}
+			{% capture versioni %}{{ page.version | split: "." | last | plus: versionmaj }}{% endcapture %}
+			{% capture latestmaj %}{{ layout.latestversion | split: "." | first | times: 100 }}{% endcapture %}
+			{% capture latesti %}{{ layout.latestversion | split: "." | last | plus: latestmaj }}{% endcapture %}
+			{% if versioni < latesti %}
+			<div class="alert alert-warning" style="margin-top:5px">
+				<p style="margin-bottom: 3px"><strong>Newer version available</strong></p>
+				<p style="margin-bottom: 0">This manual is for Foreman {{ page.version }}, but the <a href="{{ site.baseurl }}manuals/{{ layout.latestversion }}/index.html">latest version is {{ layout.latestversion }}</a>.</p>
```

Comment:
Nitpick, you use "This quickstart" on line 22 but "This manual" here, it should be the same.